lib/sinon/util/fake_server.js

```diff
@@ -33,6 +33,7 @@ function responseArray(handler) {
 }
 
 var wloc = typeof window !== "undefined" ? window.location : {};
+wloc = typeof wloc !== "undefined" ? wloc : { "host" : "localhost", "protocol" : "http"};
```

Comment:
Line 35 and 36 could be squashed into one line. You are guaranteed `that` wloc is defined.

```javascript
var wloc = typeof window !== "undefined" ? window.location : { "host" : "localhost", "protocol" : "http"};
```